tests/test_bot_slack.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import node_monitor.load_config as c
+from node_monitor.bot_slack import SlackBot  
+from unittest.mock import Mock
+
+@pytest.fixture
+def mock_slack_client(mocker):
+    mock_client = Mock()
+    mocker.patch("slack_sdk.WebClient", return_value=mock_client)
+    return mock_client
+
+def test_send_message(mock_slack_client):
+    slack_bot = SlackBot(c.TOKEN_SLACK)
+
+    expected_message = "Hello, Slack!"
+    expected_chanel = "#node-monitor"
+    slack_bot.send_message(expected_chanel, expected_message)
+
+    mock_slack_client.chat_postMessage.assert_called_once_with(
+        channel=expected_chanel,  
+        text=expected_message,
+    )
+
+
+
+@pytest.mark.live_email
+def test_send_message_slack():
+    """Send real messages to a test Slack workspace"""
+    slack_bot = SlackBot(c.TOKEN_SLACK)
+    
+    slack_channel_name = "#node-monitor"
+    message = "Hello from test_send_message_slack()"
+
+    slack_bot.send_message(slack_channel_name, message)
```

Comment:
No, not like the one that we use for email testing. We can set this up though. For end to end testing, I have a slack account set up where I check the results of this test.

Reply:
How do we set this up without running a slack client though? It would be nice to just do a `GET` request and verify that the correct info is there. Sounds like the method you use requires you to do it manually.